cmd/cluster-version-operator-tests/README.md

```diff
@@ -4,6 +4,16 @@ It integrates [openshift-tests-extension](https://github.com/openshift-eng/opens
 cluster-version-operator which allows openshift components to contribute tests to openshift-tests' suites with
 extension binaries.
 
+## Prerequisites
```

Comment:
I will continue the discussion https://github.com/openshift/cluster-version-operator/pull/1249#discussion_r2469998683 here.

My point is if any of those Prerequisites is not satisfied, the commands below will error out and then the user can act accordingly.

* if Go is not install, `make build` will fail.
* if `$KUBECONFIG` is not set, then `cluster-version-operator-tests run-suite` will fail if it is required. Moreover, perhaps not all of the tests need a `$KUBECONFIG`.